sentry-okhttp/src/main/java/io/sentry/okhttp/SentryOkHttpEvent.kt

```diff
@@ -113,7 +115,7 @@ internal class SentryOkHttpEvent(private val hub: IHub, private val request: Req
     fun startSpan(event: String) {
         // Find the parent of the span being created. E.g. secureConnect is child of connect
         val parentSpan = findParentSpan(event)
-        val span = parentSpan?.startChild("http.client.$event") ?: return
+        val span = parentSpan?.startChild("http.client.$event", "$method $url") ?: return
```

Comment:
Nice, I'm wondering if we need to do anything to make PII scrubbing work here? (I don't think so).

Reply:
Alright I just did a quick manual check. The span description definitely is processed and e.g. query params do get removed. All good!
<img width="1008" alt="image" src="https://github.com/getsentry/sentry-java/assets/1411808/1d3e9a53-b9f4-45a5-8ea8-3e4f4713f5b8">